### PR TITLE
Getting QB-Core ready for qb-jobs

### PR DESCRIPTION
**Pull request by r0adra93**
Removed duplicate locations inside of locations.lua

added QBShared.QBJobsStatus = false to jobs.lua

## Description

Removed duplicate locations inside of locations.lua

added QBShared.QBJobsStatus = false to jobs.lua

## Checklist

- [X] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [X] My code fits the style guidelines.
- [X] My PR fits the contribution guidelines.

### CODE REVIEW

**Review by Twikii — APPROVED** [review]
Good.

**Comment by KamuiKody** [comment]
Qbshared jobs has an extra table made for it that is unneeded

**Comment by r0adra93** [comment]
Which table is unneeeded?

**Comment by r0adra93** [comment]
> Qbshared jobs has an extra table made for it that is unneeded

Fixed

File: shared/jobs.lua [hunk 1]
```diff
@@ -1,5 +1,7 @@
 QBShared = QBShared or {}
 QBShared.ForceJobDefaultDutyAtLogin = true -- true: Force duty state to jobdefaultDuty | false: set duty state from database last saved
+QBShared.QBJobsStatus = false -- true: qb-jobs resource is installed | false: qb-jobs resource is not installed
```
Comment (charj) [inline shared/jobs.lua:3]: This can be done dynamically with the following native:
https://docs.fivem.net/natives/?_0x4039B485

Comment (r0adra93) [reply shared/jobs.lua:3]: But this gives the option to turn off qb-jobs for the built-in jobs and still use it for addon jobs.

Comment (charj) [reply shared/jobs.lua:3]: We can easily do a combination of the two

File: shared/jobs.lua [hunk 2]
```diff
@@ -1,6 +1,9 @@
 QBShared = QBShared or {}
 QBShared.ForceJobDefaultDutyAtLogin = true -- true: Force duty state to jobdefaultDuty | false: set duty state from database last saved
-QBShared.Jobs = {
+QBShared.QBJobsStatus = false -- true: qb-jobs resource is installed | false: qb-jobs resource is not installed
```
Comment (Z3rio) [inline shared/jobs.lua:3]:
```suggestion
QBShared.QBJobsStatus = GetResourceState("qb-jobs") ~= "missing" -- Dont touch
```

**Review by Z3rio — APPROVED** [review]
I would recommend changing this.

File: shared/jobs.lua (hunk 2 above)
Comment (r0adra93) [reply shared/jobs.lua:3]: Leave that flag as is. We don't want to automate that.

The reason is people have put a lot of work into theor cities.

It takes time to integrate everything.

They can set it to false and install the resource and still use it for
multi-job capabilities
Easy job creation for new jobs
The use of the /duty command

When they finish with the config filea for the exoatinf jobs they can switch to true to fully integrate qb-jobs to the system.

Which will reduce overall resource load and give all the other jobs the same features and capabilities that qb-jobs offers.

Comment (r0adra93) [reply shared/jobs.lua]: -- true: qb-jobs resource is installed | false: qb-jobs resource is not installed

**Pull request merged**